ansible/roles/codeready_containers_hacks/tasks/start-crc-deployment.yml

```diff
@@ -0,0 +1,39 @@
+- name: Copying over pull-secret.txt
+  become: true
+  become_user: root
+  template:
+    src: templates/pull-secret.txt.j2
```

Comment:
No need to include `templates/` when the template is in the template directory. In this case, since the template is just the secret itself, the cleanest way to do this would be:

```
- name: Copying over pull-secret.txt
  become: true
  become_user: root
  copy:
    content: "{{ codeready_containers_hacks_pull_secret_content }}"
    dest:  "{{ codeready_containers_hacks_pull_secret_path }}"
    owner: admin
    group: admin
    mode: a=rw
  tags:
  - start_crc_deployment
```